internal/feedback/feedback.go

```diff
@@ -73,7 +73,7 @@ func (cf ConstraintFeedback) LogFeedback(logger *log.Logger) {
 	if cf.Constraint != "" {
 		logger.Printf("  %v", GetUsingFeedback(cf.Constraint, cf.ConstraintType, cf.DependencyType, cf.ProjectPath))
 	}
-	if cf.LockedVersion != "" && cf.Revision != "" {
+	if cf.Revision != "" {
```

Comment:
Nm, realized I could add a test for the Any constraint to at least match my problem description.  Will do that.